apps/speed.c

```diff
@@ -252,7 +252,8 @@ const OPTIONS speed_options[] = {
     {"seconds", OPT_SECONDS, 'p',
      "Run benchmarks for specified amount of seconds"},
     {"bytes", OPT_BYTES, 'p',
-     "Run [non-PKI] benchmarks on custom-sized buffer"},
+     "Run [non-PKI] benchmarks on custom-sized buffer." 
+     " Max size is 2147483583 bytes."},
```

Comment:
I think the size here should be dropped -- it depends on INT_MAX.
If you do want this mentioned, perhaps in `doc/man1/openssl-speed.pod.in`?
E.g.:

> For a machine with a 32 bit integer, the maximum size is 2147483583 bytes.

Reply:
Moved it to the file mentioned.